examples/wearable/UIComponents/contents/user-inputs-elements/index.html

```diff
@@ -38,6 +38,10 @@ <h2 class="ui-title">User Inputs Elements</h2>
                 <li>
                     <a href="pickers/index.html">Pickers</a>
                 </li>
+
```

Comment:
Now radio.html file is located in the  contents\user-inputs-elements\controls\radio.html.
The radio, checkbox, onoff-switch is located in the contents\user-inputs-elements\controls folder. 
How about the move the file into "controls" folder  or remove the contents\user-inputs-elements\controls\radio.html, please.